entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -573,38 +578,47 @@ class EntityService(
 
     @Transactional
     fun deleteEntityAttribute(entityId: URI, expandedAttributeName: String): Boolean {
-        if (neo4jRepository.hasPropertyOfName(EntitySubjectNode(entityId), expandedAttributeName))
+        if (neo4jRepository.hasPropertyOfName(EntitySubjectNode(entityId), expandedAttributeName)) {
+            // update modifiedAt in entity if at least one attribute has been added
+            neo4jRepository.updateEntityModifiedDate(entityId)
```

Comment:
you update the modication date before being sure the attribute deletion has been done, you should only do it if the operation has suceeded